microsetta_interface/templates/sample.jinja2

```diff
@@ -16,6 +16,26 @@
     <script>
 
         $(document).ready(function(){
+            $("form").submit(function(e) {
+                var date = $("#sample_date").val()
+                date = new Date(date);
+                var lower_limit = new Date();
+                lower_limit .setFullYear( lower_limit.getFullYear() - 10 );
+                lower_limit .setDate(lower_limit.getDate());
+                lower_limit .setMonth(lower_limit.getMonth());
+                var upper_limit  = new Date();
+                upper_limit .setFullYear( upper_limit.getFullYear() );
+                upper_limit .setDate(upper_limit.getDate());
+                upper_limit .setMonth(upper_limit.getMonth() + 1);
+                if(date.getTime() < lower_limit.getTime()) {
```

Comment:
You can directly compare date objects with the < and > operators, you just can't check equality directly.
```suggestion
                if(date < lower_limit) {
```